index.ts

```diff
@@ -676,23 +740,64 @@ export default class FreeSound {
     return this.makeRequest(this.makeUri(this.uris.upload), 'POST', formData);
   }
 
+  /**
+   * Describe a previously uploaded file that does not have a description.
+   * Note: after a sound receives a description, the team of Freesound moderators
+   * still needs to process and moderate it, so it may not yet appear in Freesound.
+   * A list of sounds uploaded and described by the user, but still
+   * pending processing and moderation, is viewable with
+   * the getPendingSounds() method. This method requires OAuth2 authentication. See
+   * https://freesound.org/docs/api/resources_apiv2.html#describe-sound-oauth2-required
+   * for more information.
+   * @param description a description for an uploaded sound
+   */
   describe(description: { description: string }) {
     this.checkOauth();
     const formData = this.makeFormData(description);
     return this.makeRequest(this.makeUri(this.uris.upload), 'POST', formData);
   }
 
+  /**
+   * Retrieve a list of audio files uploaded by
```

Comment:
nit: Seems like the line break of this doc comment doesn't match with the other paragraphs. For readability it might be better to have a more consistent line break. Seems like prettier [doesn't have support for this ](https://github.com/prettier/prettier/issues/265) so we have to format manually

Reply:
Oh, I somewhat arbitrarily decided to make each documentation comment extend no longer than the block of code it describes.

I'll make it match the line breaks of other documentation comments for consistency.